MultiTable/test/SubjectGroup.js

```diff
@@ -70,7 +69,7 @@ describe('./test/SubjectGroup.js >', () => {
         isPublished: true,
         name: 'JJ1',
         parentAbsolutePath: 'Fellowship.Gandalf',
-        description: 'Updated Subjected'
+        description: 'Updated Subjected',
```

Comment:
"Subjected"?

Reply:
I don't know. I have not written this, I just fixed style issue. @iamigo